english/humbleservant/humbleservant.md

```diff
@@ -0,0 +1,2 @@
+Turns out that your humble servant is a huge mech robot with a giant drill equipped! Good work assembling that robot before. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.